graph/appender/util.go

```diff
@@ -8,13 +8,13 @@ import (
 	"github.com/kiali/kiali/graph"
 	"github.com/kiali/kiali/log"
 	"github.com/kiali/kiali/prometheus/internalmetrics"
-	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	prom_v1 "github.com/prometheus/client_golang/api/prometheus/v1"
```

Comment:
again, I see "prom" when you would think its "prometheus_v1". I think I can infer that you are restricting the name with four-characters - what happens if four characters aren't enough to make an alias unique? Then you really are going to trigger my OCD to an 11.

Reply:
Yes, OK, so prometheus_v1 just starts to get long.  Here is what I will do, I will meet you in the middle.  If 4-letters (or less) is not sufficiently clear I'll use the whole name.  I believe "prom" is fine here, so it will stay.